Formula/yacas.rb

```diff
@@ -15,18 +15,19 @@ class Yacas < Formula
 
   option "with-server", "Build the network server version"
 
-  def install
-    args = %W[
-      --disable-silent-rules
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-    ]
-
-    args << "--enable-server" if build.with? "server"
+  depends_on "cmake" => :build
+  depends_on :xcode => :build
 
-    system "./configure", *args
-    system "make", "install"
-    system "make", "test"
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-G", "Xcode", "-DENABLE_CYACAS_GUI=OFF",
+                            "-DENABLE_CYACAS_KERNEL=OFF", *std_cmake_args
+      xcodebuild "-target", "ALL_BUILD", "-project", "YACAS.xcodeproj",
+                 "-configuration", "Release", "SYMROOT=build/cyacas/libyacas"
+    end
+    bin.install "build/cyacas/libyacas/Release/yacas"
+    lib.install Dir["build/cyacas/libyacas/Release/{libyacas.a,yacas.framework}"]
+    pkgshare.install "scripts"
```

Comment:
Would be good to request a `make install` upstream but otherwise 👍. Sorry for the delay here.